foundation/features/org.jboss.tools.foundation.license.feature/feature.properties

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2010-2012 Red Hat, Inc. and others.
+# All rights reserved. This program and the accompanying materials 
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+# 
+# Contributors:
+#     JBoss by Red Hat - Initial implementation.
+##############################################################################
+# feature.properties
+# contains externalized strings for feature.xml
+# "%foo" in feature.xml corresponds to the key "foo" in this file
+# java.io.Properties file (ISO 8859-1 with "\" escapes)
+# This file should be translated.
+
+# "featureName" property - name of the feature
+featureName=JBoss Tools License feature
+
+# "providerName" property - name of the company that provides the feature
+providerName=Red Hat JBoss Middleware
```

Comment:
I believe that "Red Hat JBoss Middleware" is actually the name we have to use here.